### DISCUSSION TITLE
Add Hooks for McMMO and MMoCore

### DISCUSSION THREAD

**Discussion by Hyrulez** [BG-Software-LLC/WildTools#87] [Denied Feature Requests] [answered]
As of now players that use the WildTools items, do not get McMMO and MMOCore XP. I propose a simple hook to look for those two plugins so that players using said tools will acquire the specified exp. 

This would mainly pertain to Harvester Hoes, and trench pickaxes.

**Comment by OmerBenGera** [comment, marked as answer]
Add "mcMMO" (make sure you match the exact same name from /pl list) to the other plugins list in the config.
Same for any other plugin you want WildTools to work with.
